.circleci/config.yml

```diff
@@ -49,9 +57,19 @@ jobs:
       - run:
           name: Publish package to Artifactory
           command: npm publish
+      - run:
+          name: Announce Publish
+          command: |
+            chmod +x .circleci/announcePublish.sh
+            .circleci/announcePublish.sh "Common Logger" "$(node -e "console.log(require('./package.json').name)")@$(node -e "console.log(require('./package.json').version)")"
+      - slack/status:
+          fail_only: true
+          failure_message: "Triggered by: *${CIRCLE_USERNAME}* \n\n Ooops! here's an error log, the *$CIRCLE_JOB* job has failed! :circleci-fail:"
```

Comment:
Also, please check this message and change it to something else, or something generic like: `Ooops! The *$CIRCLE_JOB* job has failed! :circleci-fail:`